ci-operator/step-registry/baremetalds/e2e/baremetalds-e2e-workflow.yaml

```diff
@@ -10,6 +10,7 @@ workflow:
       - ref: baremetalds-e2e-conf
       - ref: baremetalds-e2e-test
     post:
+    - chain: gather
```

Comment:
Not sure if you already had it in progress, but since we have other workflows to be updated:
```
ci-operator/step-registry/baremetalds/e2e/ovn/ipsec/baremetalds-e2e-ovn-ipsec-workflow.yaml
ci-operator/step-registry/baremetalds/e2e/serial/ipv4/baremetalds-e2e-serial-ipv4-workflow.yaml
ci-operator/step-registry/baremetalds/e2e/upgrade/baremetalds-e2e-upgrade-workflow.yaml
ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml
ci-operator/step-registry/baremetalds/sno/baremetalds-sno-workflow.yaml
```

And since @ardaguclu started to refactored the steps in chains, it could be worth to modify directly the `baremetalds-ipi-post` [step](https://github.com/openshift/release/blob/master/ci-operator/step-registry/baremetalds/ipi/post/baremetalds-ipi-post-chain.yaml) and use it directly in all the relevant workflows